### ISSUE TITLE
Moderation: Filtering archived posts by topic

### ISSUE THREAD

**Issue by Miaplacidus** [hicommonwealth/commonwealth#4729]
Currently, we can (via a checkbox) include archived posts after we have chosen a topic. We can also navigate to a page that displays only archived posts. However, we also want to be able to view archived posts filtered by topic. The UI does not currently allow for this. New designs will be needed to allow for users to view archived posts filtered by topic. 

@zakhap @meeshlin

**Comment by CowMuon** [comment]
We can backlog this once design is ready.

**Comment by zakhap** [comment]
@zakhap to edit

**Comment by jnaviask** [comment]
Feature exists, ticket no longer needed.
